packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -0,0 +1,92 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import autobind from 'autobind-decorator'
+import themed from 'react-themed'
+
+@themed(['FreeDraw', 'FreeDraw_ToolTip'], { pure: true })
```

Comment:
I only see reference to 'FreeDraw_Tooltip'.
Since this style is in the `Gmap.css` perhaps it would be better as `Gmap_FreeDrawTootip`

Reply:
I agree I can change that